client/src/pages/Results/VariantDiseaseHeatmap.tsx

```diff
@@ -0,0 +1,147 @@
+    return label.length > maxLength ? label.slice(0, maxLength) + '…' : label
+  }
+
+  // must appease Typescript overlords
+  const x = (i: number) => xScale(i)!
+  const y = (i: number) => yScale(i)!
+
+  return (
+    <div style={{ position: 'relative' }}>
+      <svg width={width} height={height}>
+        <Group top={margin.top} left={margin.left}>
+          <HeatmapRect
+            data={columns}
+            xScale={x}
+            yScale={y}
+            colorScale={colorScale}
+            binWidth={xScale.bandwidth()}
+            binHeight={yScale.bandwidth()}
+            gap={2}
+          >
+            {(heatmapData) =>
+              heatmapData.map((columnCells, columnIndex) =>
+                columnCells.map((cell, rowIndex) => {
+                  const cellBin = cell.bin as { count: number }
+                  const value = cellBin.count
+                  if (value === 0) return null
+
+                  return (
+                    <rect
+                      key={`${columnIndex}-${rowIndex}`}
+                      x={cell.x}
+                      y={cell.y}
+                      width={cell.width}
+                      height={cell.height}
+                      fill={colorScale(value)}
+                      onMouseEnter={() =>
+                        showTooltip({
+                          tooltipLeft: cell.x + margin.left,
+                          tooltipTop: cell.y + margin.top,
+                          tooltipData: {
+                            value,
+                            variant: variants[rowIndex],
+                            disease: diseases[columnIndex],
+                          },
+                        })
+                      }
+                      onMouseLeave={hideTooltip}
+                    />
+                  )
+                }),
+              )
+            }
+          </HeatmapRect>
+
+          {/* Y axis: variants (rows) */}
+          <AxisLeft
+            scale={yScale}
+            tickFormat={(i) => truncateLabel(variants[i])}
+            tickValues={variants.map((_, i) => i)}
+            tickLabelProps={() => ({
+              fontSize: 11,
+              textAnchor: 'end',
+              dy: '0.33em',
+            })}
+          />
+
+          {/* X axis: diseases (columns) */}
+          <AxisBottom
+            scale={xScale}
+            top={yMax}
+            tickFormat={(i) => truncateLabel(diseases[i])}
+            tickValues={diseases.map((_, i) => i)}
+            tickLabelProps={() => ({
+              fontSize: 11,
+              angle: -45,
+              textAnchor: 'end',
+            })}
+          />
+        </Group>
+      </svg>
+
+      {tooltipData ? (
+        <Tooltip top={tooltipTop} left={tooltipLeft} style={tooltipStyles}>
+          <strong>{tooltipData.variant}</strong> → <strong>{tooltipData.disease}</strong>
+          <div>{tooltipData.value} supporting evidence</div>
```

Comment:
I think this should be just "evidence item(s)", "item(s)", or ["items" if value > 1 else "item"]